runtime/near-vm-runner/src/cache.rs

```diff
@@ -85,6 +87,140 @@ impl fmt::Debug for MockCompiledContractCache {
     }
 }
 
+/// A cache that stores precompiled contract executables in a directory of a filesystem.
+///
+/// This directory can optionally be a temporary directory. If created with [`Self::test`] the
+/// directory will be removed when the last instance of this cache is dropped.
+///
+/// Clones of this type share the same underlying state and information. The cache is thread safe
+/// and atomic.
+///
+/// This cache however does not implement any clean-up policies. While it is possible to truncate
+/// a file that has been written to the cache before (`put` an empty buffer), the file will remain
+/// in place until an operator (or somebody else) removes files at their own discretion.
```

Comment:
I was just going to ask you about this. Why did you decide to not clean up the cache? Do you expect it to be growing slow enough to ever actually cause trouble? Do you know what would be the total size of cache today?

Reply:
This is again something that the previous rocksdb based implementation does as well. You can check the corresponding column in rocksdb for the current numbers. I have no clue how to do that myself.

The size of this directory should end up being in the same ballpark in the long term.